frontstage/views/surveys/help/surveys_help.py

```diff
@@ -9,10 +9,54 @@
 
 from frontstage.common.authorisation import jwt_authorization
 from frontstage.controllers import survey_controller, conversation_controller
-from frontstage.models import HelpOptionsForm, HelpCompletingThisSurveyForm, SecureMessagingForm
+from frontstage.models import HelpOptionsForm, HelpCompletingThisSurveyForm, SecureMessagingForm, \
+    HelpInfoAboutThisSurveyForm
 from frontstage.views.surveys import surveys_bp
 
 logger = wrap_logger(logging.getLogger(__name__))
+help_completing_this_survey_title = "Help completing this survey"
+info_about_this_survey_title = "Information about this survey"
+option_template_url = {
+        'help-completing-this-survey': 'surveys/help/surveys-help-completing-this-survey.html',
+        'info-about-this-survey': 'surveys/help/surveys-help-info-about-this-survey.html'
+}
+sub_option_template_url = {
+        'do-not-have-specific-figures': 'surveys/help/surveys-help-specific-figure-for-response.html',
+        'unable-to-return-by-deadline': 'surveys/help/surveys-help-return-data-by-deadline.html',
+        'exemption-completing-survey': 'surveys/help/surveys-help-exemption-completing-survey.html',
+        'why-selected': 'surveys/help/surveys-help-why-selected.html',
+        'time-to-complete': 'surveys/help/surveys-help-time-to-complete.html',
+        'how-long-selected-for': 'surveys/help/surveys-help-how-long-selected-for.html',
+        'penalties': 'surveys/help/surveys-help-penalties.html',
+        'info-something-else': 'surveys/help/surveys-help-info-something-else.html'
+}
+subject_text = {
```

Comment:
Same as above.  `subject_text_mapping` maybe? Just something to imply the variable contains many of something rather then just 1 string.

Reply:
makes sense.. will change it